### PR TITLE
Modernize project to Ember 3.5

### PR DESCRIPTION
**Pull request by loganrosen**
I've modernized `ember-cli-welcome-page` to Ember 3.5, using `ember-cli-update` in the process. This involved, other than the changes to the default files, reworking the integration test to use `ember-test-helpers` instead of jQuery and making some style changes to `welcome-page.hbs` to pass the new template linting.

### CODE REVIEW

**Comment by locks** [comment]
I see some linting errors @loganrosen, could you address them?

**Comment by loganrosen** [comment]
I fixed the linting issues, largely by removing the `node-tests/` directory, which contained unused Mocha tests (Mocha doesn't even appear in `devDependencies`. Please let me know if you think these should be kept.

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017
+Copyright (c) 2018
```
Comment (rwjblue) [inline LICENSE.md]: Lets make this a range instead

File: README.md [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
+* `yarn`
 
-## Running
+### Linting
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+* `yarn lint:hbs`
+* `yarn lint:js`
+* `yarn lint:js -- --fix`
```
Comment (rwjblue) [inline README.md]: I believe this should be `yarn lint:js --fix` (The double dash separating the arg is only needed for npm).

File: package.json [hunk 3]
```diff
@@ -5,40 +5,51 @@
...
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "ember test",
+    "test:all": "ember try:each"
   },
   "dependencies": {
     "broccoli-funnel": "^1.0.1",
-    "ember-cli-babel": "^6.6.0",
-    "ember-cli-htmlbars": "^2.0.3"
+    "ember-cli-babel": "^6.16.0",
```
Comment (rwjblue) [inline package.json]: Can you bump this to ^7.1.3?

**Comment by rwjblue** [comment]
@loganrosen - Mind rebasing and fixing up the conflicts?

**Comment by locks** [comment]
I was trying to lend a hand and it seems I goofed it up 😓

**Pull request closed without merging**